test/e2e-tests.sh

```diff
@@ -59,6 +59,5 @@ initialize $@
 
 header "Running tests for Knative serving $KNATIVE_VERSION"
 
-go_test_e2e ./test/e2e || fail_test
-
+go_test_e2e -timeout=30m ./test/e2e || fail_test
```

Comment:
Interesting. You did this because you've seen the test take upward of 30 minutes? Or are you just being cautious? A 30 minutes test is not a good thing obviously...

Reply:
There was a timeout for going beyond default value, 10m. I kept it this 3 times to be sure.